internal/component/configmap/etcdconfig.go

```diff
@@ -22,27 +22,22 @@ const (
 	defaultInitialClusterToken     = "etcd-cluster"
 	defaultInitialClusterState     = "new"
 	// For more information refer to https://etcd.io/docs/v3.4/op-guide/maintenance/#raft-log-retention
-	// TODO: Ideally this should be made configurable via Etcd resource as this has a direct impact on the memory requirements for etcd container.
-	// which in turn is influenced by the size of objects that are getting stored in etcd.
-	defaultSnapshotCount = 75000
+	defaultSnapshotCount = int64(75000)
+	commTypePeer         = "peer" // communication type
+	commTypeClient       = "client"
```

Comment:
```suggestion
	advURLTypePeer   = "peer"
	advURLTypeClient = "client"
```

Reply:
can we just use clear names and just call it `advertiseURLTypePeer` and `advertiseURLTypeClient` instead?